libnavigation-base/src/main/java/com/mapbox/navigation/base/options/NavigationOptions.kt

```diff
@@ -7,31 +7,33 @@ import com.mapbox.navigation.base.typedef.RoundingIncrement
 import com.mapbox.navigation.base.typedef.TimeFormatType
 
 /**
- * Default navigation pooling delay
+ * Default approximate prediction interval lag in milliseconds
+ *
+ * This value will be used to offset the time at which the current location was calculated
+ * in such a way as to project the location forward along the current trajectory so as to
+ * appear more in sync with the users ground-truth location
  */
-const val DEFAULT_NAVIGATOR_POLLING_DELAY = 1500L
+const val DEFAULT_NAVIGATOR_PREDICTION_INTERVAL_LAG = 1500L
 
 /**
  * Defines navigation options
  *
  * @param roundingIncrement defines the increment displayed on the instruction view
  * @param timeFormatType defines time format for calculation remaining trip time
- * @param navigatorPollingDelay defines approximate location engine interval lag in milliseconds
+ * @param navigatorPredictionIntervalLag defines approximate prediction interval lag in milliseconds
  *
  * This value will be used to offset the time at which the current location was calculated
  * in such a way as to project the location forward along the current trajectory so as to
  * appear more in sync with the users ground-truth location
  *
- * @param fasterRouteDetectorInterval defines time interval in milliseconds for detection is faster route available
- * @param distanceFormatter [DistanceFormatter] for format distances showing in notification during navigation
```

Comment:
should keep `distanceFormatter` description?

Reply:
Great catch!